dotnetv3/S3/scenarios/TransferUtilityBasics/TransferUtilityBasics/TransferBasics.cs

```diff
@@ -0,0 +1,155 @@
+﻿// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier:  Apache-2.0
+
+// snippet-start:[S3.dotnetv3.TransferUtilityBasics.Main]
+
+// The Amazon S3 client here uses the default user credentials
```

Comment:
// This Amazon S3 client uses the default user credentials

Reply:
changed